### ISSUE TITLE
Maintainer request

### ISSUE THREAD

**Issue by BhawaniSingh** [jhipster/jhipster-devbox#105]
Hello
I would like to maintain this project, I've been using jhipster since gulp time, so I would like to contribute to the project

**Comment by jdubois** [comment]
Thanks a lot @BhawaniSingh !

Is it fine for you if we create a specific GitHub team for the Devbox, so you (and maybe other interested people) can have write access to the code? Then I need to check how I can give you permissions on Vagrant cloud.

**Comment by BhawaniSingh** [comment]
Yeah that would be great, the issue I'm facing currently is that on bento ubuntu image, certain packages are not getting installed properly, 
```
Errors were encountered while processing:

ubuntu-release-upgrader-core
ubuntu-release-upgrader-gtk
update-manager-core
update-notifier-common
ubuntu-server
update-notifier
update-manager
xubuntu-desktop
```
Is this the issue or is there any other issue. (I tried 4-5 times and every time I'm getting the same issue)

**Comment by BhawaniSingh** [comment]
@jdubois Is this something to expect?

![jhipster-devbox-6](https://user-images.githubusercontent.com/2702715/57175662-218e2d80-6e6c-11e9-8d4a-7c88ce48f61a.png)

**Comment by jdubois** [comment]
Yes! But I also had several packages errors, I wonder if we shouldn't move away from Bento if it's not stable enough.

**Comment by BhawaniSingh** [comment]
I replaced base image "bento/ubuntu-18.04" with "ubuntu/bionic64".

**Comment by jdubois** [comment]
@BhawaniSingh oh that's awesome!!

**Comment by cbornet** [comment]
Let's hope the new official Ubuntu box is not as crappy as the previous one

**Comment by PierreBesson** [comment]
@BhawaniSingh Thank you for stepping in as maintainer !

**Comment by jdubois** [comment]
Awesome -> @BhawaniSingh I created a "Devbox maintainers" team with write access to the repo, and invited you to it. Welcome to the team!

**Comment by BhawaniSingh** [comment]
@jdubois Thanks, Its an honor to be the part of the team :)
